Automattic/wp-calypso
# Stepper Tracking: Simplify `calypso_new_user_site_creation`

Fixes https://github.com/Automattic/wp-calypso/issues/96724

## Proposed Changes

Simplify the logic to calculate `calypso_new_user_site_creation`.
It will be triggered only if the user has just been created and the site is new.

## Why are these changes being made?


Too much complexity in the calculation may lead to problems in retrieving the right data.

## Testing Instructions



- Go through /start and /setup/onboarding
- Verify that `calypso_new_user_site_creation` is triggered when using a new user, and it is not when using an already existing one.
